extensions/common/iam/oauth2/oauth2-core/src/main/java/org/eclipse/edc/iam/oauth2/Oauth2ServiceExtension.java

```diff
@@ -167,8 +169,22 @@ private Oauth2ServiceConfiguration createConfig(ServiceExtensionContext context)
                 .privateKeyResolver(privateKeyResolver)
                 .certificateResolver(certificateResolver)
                 .notBeforeValidationLeeway(context.getSetting(NOT_BEFORE_LEEWAY, 10))
+                .issuedAtValidationLeeway(getIssuedAtValidationLeeway(context))
                 .tokenExpiration(TimeUnit.MINUTES.toSeconds(tokenExpiration))
                 .build();
     }
 
+    private int getIssuedAtValidationLeeway(ServiceExtensionContext context) {
+        if (!context.getConfig().hasKey(ISSUED_AT_LEEWAY)) {
+            String message = "No value was configured for '%s'.".formatted(ISSUED_AT_LEEWAY) +
```

Comment:
Please use `var', do not concatenate, and shorten the message:

format("No value was configured for %s. Consider setting a leeway of 2-5s in production to avoid problems with clock skew", ....);

Reply:
fixed